framework/math/ParallelVector/ParallelSTLVector.cc

```diff
@@ -56,6 +58,13 @@ std::unique_ptr<ParallelVector> ParallelSTLVector::MakeClone() const
 double* ParallelSTLVector::Data() { return values_.data(); }
 const double* ParallelSTLVector::Data() const { return values_.data(); }
 
+const std::vector<double>& ParallelSTLVector::LocalSTLData() const
```

Comment:
This function is not overriding anything in the base class. It is a pretty efficient way of getting to the local data if you know you are dealing with a ParallelSTLVector